cmd/users.go

```diff
@@ -496,6 +496,24 @@ var RemoveUserFromOrganization = &cobra.Command{
 	},
 }
 
+var resetPasswordCmd = &cobra.Command{
+	Use:   "resetPassword",
```

Comment:
To fit with the rest of the commands, the styling is kebab-case. Also worth adding in some aliases too
```
Use:   "reset-password",
Aliases: []string{"reset-pass","reset","rp"},
```

Reply:
I didn't think `reset` on it's own made sense?

Reply:
I'm happy to omit `reset` but based on where it lives in the command set under `user` would there be another `reset`?

Happy with whatever you decide to do with it

Reply:
> based on where it lives in the command set under `user`

It only lives there in the code, in actual usage nothing lives under user. Things only live under `list`, `add`, `delete`, etc. I don't really like the way our current `verb noun` system works because where do you put "actions?" So it's at the root level, `lagoon reset-password`, and in that case `lagoon reset` doesn't make any sense?

If we used `noun verb` instead (like `lagoon user add`), then I agree a `reset` could make sense. That's also why `deploy` is a root command, `lagoon deploy` instead of `lagoon environment deploy`. But that's a discussion for v2 :)

Reply:
All good, remove it then